app/assets/stylesheets/autocomplete.css

```diff
@@ -6,23 +6,19 @@ ul.ui-autocomplete {
   margin: 0;
   padding: 0;
   border: solid 1px #999;
-  cursor: default; 
+  cursor: pointer;
 }
 
 ul.ui-autocomplete li {
   background-color: #FFF;
   border-top: solid 1px #DDD;
   margin: 0;
-  padding: 0; 
+  padding: 3px !important; 
 }
 
-ul.ui-autocomplete li a {
-  color: #000;
-  display: block;
-  padding: 3px; 
-}
-ul.ui-autocomplete li a.ui-state-hover, ul.ui-autocomplete li a.ui-state-active {
-     background-color: #FFFCB2; 
+ul.ui-autocomplete li.ui-state-focus {
+     background-color: #337ab7; 
```

Comment:
Can we use SASS (e.g. a variable) for this instead of hard-coding a number that we'll have to remember to come back and change if we ever change our UI / color scheme?